libraries/AP_Scripting/applets/Aerobatics/FixedWing/plane_aerobatics.lua

```diff
@@ -1812,6 +2190,168 @@ function calculate_rudder_offset(ahrs_quat, ahrs_gyro, airspeed_constrained)
    return rudder_ofs
 end
 
+--[[
+   convert a table of bytes to a lua string
+--]]
+function bytes_to_string(bytes)
+   local ret = {}
+   for _, b in ipairs(bytes) do
+      if b == 0 then
+         break
+      end
+      table.insert(ret, string.char(b))
+   end
+   return table.concat(ret)
+end
+
+--[[
+   a lua implementation of the jitter correction algorithm from libraries/AP_RTC
+
+   note that the use of a 32 bit float lua number for a uint32_t
+   milliseconds means we lose accuracy over time. At 9 hours we have
+   an accuracy of about 1 millisecond
+--]]
+local function JitterCorrection(_max_lag_ms, _convergence_loops)
```

Comment:
This would be a great start to a new `libraries` folder you could then use require to add use it here.

Reply:
until we have mavlink support for scripts requesting libraries be uploaded with MAVFtp this would make it too hard to install